src/shape/custom_shapes.js

```diff
@@ -650,17 +650,15 @@ class Shape {
     } else if (original instanceof Vector) {
       return new Vector(queue.shift(), queue.shift(), queue.shift());
     } else if (original instanceof Color) {
+      // NOTE: Not sure what intention here is, `Color` constructor signature
+      // has changed so needed to be reviewed
       const array = [
         queue.shift(),
         queue.shift(),
         queue.shift(),
         queue.shift()
       ];
-      return new Color(
-        array.map((v, i) => v * original.maxes[original.mode][i]),
-        original.mode,
-        original.maxes
-      );
+      return new Color(original);
     }
```

Comment:
@davepagurek I'm not sure what this is meant to do but the signature of `Color` constructor is now relatively simplified. If the intention is to make a copy, pass the color object to the constructor will do so.

Reply:
I'm guessing the visual test is failing because of this change.

Reply:
oh yeah, so this is part of some code that interpolates values. It's kind of doing a lerpColor, but the color is just one of multiple properties (that vary dynamically) that get interpolated, so it serializes everything to an array of numbers, lerps all of them, then turns them back into their constituent parts.

So this code was previously trying to make a new color with the maxes as the original, but with the lerped outputs from before. What would be the best way to go about this now?

Reply:
that looks right to me, thanks!